fmriprep/workflows/fieldmap/se_pair_workflow.py

```diff
@@ -101,41 +97,6 @@ def se_pair_workflow(name='Fieldmap_SEs', settings=None):  # pylint: disable=R09
 
     return workflow
 
-def fieldmap_to_phasediff(name='Fieldmap2Phasediff'):
```

Comment:
Ok no problem, it is true it gets too long.

Regarding a more object oriented approach, I am totally in favor of that. This is why I started a never ended PR for nipype with ```InterfacedWorkflows``` that, in practice, solved this problem. I see a lot of code duplicated from that PR here if you decide to dive into that. Feel free to look at it first.